src/Struct/Config.php

```diff
@@ -63,6 +65,9 @@ class Config extends Struct
     /** @var string */
     private $filterPosition;
 
+    /** @var string */
+    private $mainVariant;
```

Comment:
Also add this to the constant `ALLOW_FOR_SERIALIZATION`.

Reply:
Already exists.